### PR TITLE
Standardised component pages

### PR DESCRIPTION
**Pull request by tomyems**
As a result of the [discussion about a template for component pages](https://github.com/UKHomeOffice/design-system/discussions/430), a pull request to standardise the format of the component pages in the Home Office design system.

### CODE REVIEW

**Comment by tomyems** [comment]
https://6304ccddfb22d77dee029268--hods.netlify.app/components

**Review by daniel-ac-martin — APPROVED** [review]
LGTM.
Does it need any more content review?

File: components/status-message/spec/StatusMessage.stories.mdx [hunk 1]
```diff
@@ -69,9 +69,9 @@ The status message is used by:
   - Returns and movement
   - Passport renewals
   - Passport application search
-  - an internal case management system
+  - a service used by staff to assess risk at the border
 
-For an internal service, this was used at the top of the screen of a case management system to:
+For an service to assess border risk, this was used at the top of the screen to:
```
Comment (Rhodine-orleans-lindsay) [inline components/status-message/spec/StatusMessage.stories.mdx:74]: small typo - should be 'For **a** service...'

**Review by Rhodine-orleans-lindsay — COMMENTED** [review]
looks good, just the small typo I highlighted @tomyems

**Comment by tomyems** [comment]
https://631846738c07707e7fc0cdfc--hods.netlify.app/components

File: components/status-message/spec/StatusMessage.stories.mdx (hunk 1 above)
Comment (tomyems) [reply components/status-message/spec/StatusMessage.stories.mdx]: Thanks! - fixed :)

**Comment by tomyems** [comment]
> LGTM. Does it need any more content review?

I would show it on Friday and then merge if everyone's happy - most of the group have seen it

**Review by Rhodine-orleans-lindsay — APPROVED** [review]

**Pull request merged**